pandas/tests/io/test_excel.py

```diff
@@ -2556,3 +2556,128 @@ def test_excelwriter_fspath(self):
         with tm.ensure_clean('foo.xlsx') as path:
             writer = ExcelWriter(path)
             assert os.fspath(writer) == str(path)
+
+
+@td.skip_if_no('odf')
+class TestODFReader(SharedItems):
```

Comment:
Hey @detrout ! So we just went through a drastic overhaul of the test cases. You will want to definitely merge master in before continuing on this which should (hopefully) make adding these tests easier.

Rather than having everything in one large `test_excel` file you'll now see an excel subfolder where that used to be. You will want to do add this as `test_odf` as a separate module to start. We'll also have to figure out how to add this into the standard `test_readers` class but let's just take care of the former here and we can go from there